README.md

```diff
@@ -81,35 +72,24 @@ python google_test_runner.py
 ## Command line options
 
 ```python
-python custom_runner.py --env [browserstack_environments] --tests [tests_to_run] --browserstack [username:token]
-
---env direct # runs tests on browserstack combinations
-             # passed to script through BROWSERSTACK environment variable (json)
+--platform web         # define platform on which run tests (currently supported: web, mobile, generic)
+--environmnet local    # define environment in which run tests (currently supported: local, browserstack, appium)
+--test_directory tests # define directory where to lookup for tests (project_root + test_directory)
 
-os.environ['BROWSERSTACK'] = '{"test_suite": [{"browser": "Firefox", "browser_version": "27.0",
-                             ' "os": "Windows", "os_version": "7", "resolution": "1024x768"},' \
-                             ' {"browser": "IE", "browser_version": "10.0", "os": "Windows",' \
-                             ' "os_version": "7", "resolution": "1024x768"}]}'
+# supported platform/environment combinations:
+#   generic/local
+#   web/local
+#   web/browserstack
+#   mobile/appium (can run on local/remote appium server or on saucelabs)
```

Comment:
Please add `generic/remote` and `web/remote` (since remote_module branch was already merged in)